File: Day_0903/src/Exam_04.java
public class Exam_04 {

	public static void main(String[] args) {
//		test(5);
		func(5);
	}

//	static int test(int n) {
//		int s=0;
//		for (int i = 1; i <= n; i++) {
//			s += i;
//		}
//		return s;
//	}

	static void test(int n) {
		for (int i = 1; i <= n; i++) {
			System.out.println("Hello World");
		}
	}

	// Àç±Í È£Ãâ
	static void func(int n) {
		if (n == 0) {
			return;
		}
		
		System.out.println("Hello World");
		func(n - 1);
	}

}

File: Day_1001/exam04.py
# Data Type

'''
    1) 단일값 저장 : integer, float, string, boolean, Complex(복소수)
    
    2) 다중 값 저장 : list[], tuple(), dictionary{}, set{}
'''

# 문자열 인덱싱[0], 슬라이싱[1:5]

str = 'Hello World!' # index : [0] ~ [11] /  [-1] ~ [-12]
print('str = ', str)
print("str[0] = ", str[0])
print("str[-12] = ", str[-12])
# str[0] = '\'

# 슬라이싱

print('str[:] = ', str[:])      # 전체
print('str[6:] = ', str[6:])    # 6 ~ 끝
print('str[2:5] = ', str[2:6])  # 2 ~ 6전까지
print('str[0:] = ', str[0:])    # 처음부터 5전까지
print('str[::2] = ', str[::2])  # 전체를 2씩 증가하면서


